docs/AppSettings.md

```diff
@@ -8,7 +8,7 @@ Name | Type | Description | Notes
 **useAnimalNames** | **Boolean** | Flag specifying whether animal names should be used for anonymous users. |  [optional]
 **conversationRetentionSeconds** | **Integer** | Number of seconds of inactivity before a conversation’s messages will be automatically deleted. |  [optional]
 **echoPostback** | **Boolean** | A boolean specifying whether a message should be added to the conversation history when a postback button is clicked. |  [optional]
-**ignoreAutoConversationStart** | **Boolean** | A boolean specifying whether a non message event coming from a channel will trigger a start conversation event and count as an active user conversation (AUC). Not applicable when using the SDK. |  [optional]
+**ignoreAutoConversationStart** | **Boolean** | A boolean specifying whether a non message event coming from a social channel will trigger a start conversation event and count as an active user conversation (AUC).<br>**NOTE:** Use of the `startConversation()` or `startConversationWithCompletionHandler` methods from a Web/iOS/Android SDK Client will NOT be blocked by this setting. |  [optional]
```

Comment:
`startConversationWithCompletionHandler()`

Reply:
Does `Client` need to be capitalized?

Reply:
Do we also need to capitalize NOT?

Reply:
I think we can omit `Client` altogether. I also think not should not be capitalized.